components/globals/Input/Input.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
```

Comment:
Is this a straight copy or?

Asking as I would rather make these types of edits outside of the current PR.

Reply:
Yup, the only edit is on line 76 adding the meta.touched. Makes sense, though first, here was my logic. 
I'd like to add on blur validation. I need to add a dirty (touched) check to the form components. I need to edit the Input component. I'm afraid to modify the forms-form components. I guess I should "clone" the input and start customizing it. Maybe I should do this later? Nah, I need it now :)

But yes, this small change could be added to the forms-form TextInput component instead if you'd prefer?

Reply:
Wondering if we use the existing TextInput and update the `ErrorMessage` to take a meta prop

We could check for meta.touched at that level and possibly do something with the `validateOnBlur`

Reply:
Sounds good. I think I'm over my fear of touching forms-form components now. Adding the meta.touched field should really only impact form behavior if the validateOnBlur is set to true.

Though in the future, maybe over the Portage. I'd (/whoever) really like to add basic elements like Select / Input / .. that we can customize for the app. I'm worried one day we'll accidentally break the forms-form components